Expensify/App
# Changed the Amount Text to TextInput

### Details
- Allow pasting of amount in `IOUAmount` field


### Fixed Issues

$ https://github.com/Expensify/App/issues/6285

### Tests
1. Checked by pasting numeric and decimal values
2. Checked that alphabets shouldn't be allowed
3. Background and foreground switching



### QA Steps
1. Copy a numeric value
2. Go to FAB and click on Send money or Request money option
3. Paste the amount and you should be able to paste the amount


### Tested On


### Screenshots


#### Web


#### Mobile Web

<img width="343" alt="mweb-amount-pasting" src="https://user-images.githubusercontent.com/57435789/146628522-b06a9fbe-eb50-4be5-9293-37958aeca7b1.png">

#### Desktop


#### iOS

https://user-images.githubusercontent.com/57435789/146578578-8787ae17-00fe-411d-b13c-8e8e31c5822a.mov

#### Android

<img width="302" alt="android-amount-pasting" src="https://user-images.githubusercontent.com/57435789/146578654-acc79bdb-e97b-41fa-8461-6e4e9f9aef91.png">
